librustzcash/src/rustzcash.rs

```diff
@@ -50,9 +50,12 @@ use zcash_primitives::{
         fs::{Fs, FsRepr},
         FixedGenerators, JubjubEngine, JubjubParams, PrimeOrder, ToUniform, Unknown,
     },
+    //merkle_tree::CommitmentTreeWitness,
```

Comment:
Why is this commented?

Reply:
they renamed CommitmentTreeWitness to MerklePath in master between the UDA fork and the current master. as far as I can tell, no changes in behavior, just a name change